domain/src/main/java/org/oppia/android/domain/profile/ProfileManagementController.kt

```diff
@@ -237,7 +237,7 @@ class ProfileManagementController @Inject constructor(
         dateCreatedTimestampMs = oppiaClock.getCurrentTimeMs()
         this.isAdmin = isAdmin
         readingTextSize = ReadingTextSize.MEDIUM_TEXT_SIZE
-        appLanguage = AppLanguage.ENGLISH_APP_LANGUAGE
+        oppiaLanguage = OppiaLanguage.ENGLISH
```

Comment:
I left this since i was not able to find any suitable way to fix Tests timeout issue which was being caused by combining data from both TranslationController and ProfileManagementController. The only resort was to update language here so as to access the same while displaying the Options list to prevent combining data which was causing issue with tests timing out.

Reply:
AppLanguageSelection contains the selection_type_case which is used during computing what language should be displayed by TranslationController.

Reply:
We shouldn't be changing production values due to test behaviors, we should instead be fixing the tests. What specific problems did you run into?

Reply:
The language on "options" screen was not in sync with the currently selected language on the "select language" screen.

Reply:
That sounds like your implementation isn't complete then. The translation controller should be the sole source of truth for current language to be used everywhere in the app. In fact, I suggest going ahead and removing the functionality from here & the profile proto tied to app language. Any failures that come from that require fixing and indicate a possible bug in the production implementation.

Reply:
Have updated this. Thanks.